core/src/test/scala/com/raphtory/algorithms/MaxFlowTest.scala

```diff
@@ -18,8 +18,9 @@ class MaxFlowTest extends BaseCorrectnessTest {
   test("test on two connected cliques") {
     correctnessTest(
             TestQuery(MaxFlow[Long]("2", "101", maxIterations = 10000), 9899),
-            "MaxFlow/bottleneck.csv",
             "MaxFlow/bottleneckResult.csv"
     )
   }
+
+  override def setSource(): Source = CSVEdgeListSource(ResourceSpout("MaxFlow/bottleneck.csv"))
```

Comment:
This is naughty. The test file actually has weights but the result is the same if we set all weights to 1. You actually changed the test though the result is the same. You should change `setSource()` to use the `WeightedGraphBuilder`.

Reply:
my bad 🥲